standalone/pruntime/enclave/src/lib.rs

```diff
@@ -1327,10 +1356,14 @@ fn get_info(_input: &Map<String, Value>) -> Result<Value, Value> {
     let local_state = LOCAL_STATE.lock().unwrap();
 
     let initialized = local_state.initialized;
-    let pubkey = local_state
-        .identity_key
-        .as_ref()
-        .map(|pair| hex::encode(&pair.public()));
+    let genesis_block_hash = match &local_state.genesis_block_hash {
+        Some(genesis_block_hash) => hex::encode(genesis_block_hash.as_ref()),
+        None => "".to_string(),
+    };
+    let pubkey = match &local_state.identity_key {
+        Some(pair) => hex::encode(pair.public().as_ref()),
+        None => "".to_string(),
+    };
```

Comment:
```suggestion
    let genesis_block_hash = local_state.genesis_block_hash.as_ref().map(|hash| hex::encode(hash)) {;
    let pubkey = local_state.identity_key.as_ref().map(|pair| hex::encode(pair.public().as_ref()));
```

Then we can return `Option<String>` instead of `String` in the response

Reply:
fixed